### ISSUE TITLE
Base thresholds for each models

### ISSUE THREAD

**Issue by MinseokCHAE** [serengil/deepface#423]
Hi Serengil.
First of all I appreciate for your work. It really helps me a lot.
By the way, I'm new to your deepface library, so I want to ask a question.

/deepface/DeepFace.py  (from line 197 to 202)
In the function 'verify', you set the identification true or false by comparing distance and threshold which is set again by 'findThreshold' function in line 197.
And when I go to /deepface/commons/distance.py where the function 'findThreshold' is defined in line 30, I found that there are base thresholds for each models.

I wonder if they are just recommended values or are there any objective criteria according to the Papers, especailly threshold for model Facenet's cosine distance; 0.4.

By knowing the origin of these thresholds, I become to be able to explain others why I judged two faces are identifed true or false by setting the threshold 0.4, in other words, simililarity 60%(Strictly, it does not mean similiarity is 60%, but anyway...).

In summary, I need the clear criteria of the value of threshold.

Thank you for your help.

**Comment by serengil** [comment]
You can find out more on this [tutorial](https://sefiks.com/2020/05/22/fine-tuning-the-threshold-in-face-recognition/)
